package.json

```diff
@@ -40,14 +41,24 @@
       "types": "./layouts.d.ts"
     },
     "./formik": {
-      "import": "./dist/formik.js",
-      "require": "./dist/cjs/formik.js",
+      "import": "./dist/formik/index.js",
+      "require": "./dist/cjs/formik/index.js",
```

Comment:
<img width="617" alt="Screenshot 2024-04-09 at 10 37 31 AM" src="https://github.com/bigbinary/neeto-ui/assets/26121158/71eca30a-063a-4921-835e-1c8574593126">

Generated file inside formik directory is `index.js.js`, but here we are expecting it to be `index.js`. Could you please check why it is being coming as `index.js.js`?"